client/components/avatar/UserAvatarEditor.js

```diff
@@ -40,6 +42,10 @@ export function UserAvatarEditor({ currentUsername, username, setAvatarObj, sugg
 	const [clickUpload] = useFileInput(setUploadedPreview);
 
 	const clickUrl = () => {
+		if (avatarFromUrl === '') {
+			toastr.error('Please Enter a URL');
+			throw new Meteor.Error();
+		}
```

Comment:
We don't use `Meteor` and `toastr` directly on React components, we have a hook for `useToastMessageDispatch` for this. Btw, I think you should only disable the button, instead of showing a toast.

Reply:
I removed the toast and disabled the button now! Please review it :D

Reply:
@im-adithya I think just stop the event isn't a nice user experience. We need to give some affordance to the user. Maybe we should use the hook that @gabriellsh mentioned or disable the action button if the URL field is empty.

Reply:
Yes, @dougfabris, I've done the same; the test has failed due to the same regex error here as well.